src/unum/wireless/wireless_nl80211_lede.c

```diff
@@ -0,0 +1,25 @@
+#include "unum.h"
+// All functions dependent on LEDE specific platform
+// Given phy name, get the first interface (VAP) name
+// Since the phy to interface mapping is maintained in wireless_iwinfo,
+// we use iwinfo APIs
+char*
+wt_platform_nl80211_get_vap(char *phyname)
```

Comment:
Please keep the return type on the same line as the function name.

Reply:
Done.